### PR TITLE
doc: default role for first time OIDC user login, SH-1032

### PR DESCRIPTION
**Pull request by Isammoc**

### CODE REVIEW

**Review by slandelle — COMMENTED** [review]
Could you also please update the release note?

File: self-hosted-legacy/release-notes/1.23.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Gatling Enterprise 1.23
+menutitle: What's New in 1.23
+seotitle: What's New in Gatling Enterprise 1.23
+description: Learn about the main new features of Gatling Enterprise 1.23
+lead: Learn about the main new features of Gatling Enterprise 1.23
+date: 2024-09-09T11:00:00+02:00
```
Comment (Isammoc) [inline self-hosted-legacy/release-notes/1.23.md:7]: I don't know what to put here (for now)

Comment (slandelle) [reply self-hosted-legacy/release-notes/1.23.md:7]: Use today's date

File: self-hosted-legacy/install/server/manual/index.md [hunk 2]
```diff
@@ -525,11 +531,12 @@ oidc {
   scopes = ["email", "profile"]
   jwksRefreshFrequency = 1440
   mapping {
-    username: "email"
-    firstname: "given_name"
-    lastname: "family_name"
-    email: "email"
+    username = "email"
```
Comment (slandelle) [inline self-hosted-legacy/install/server/manual/index.md:534]: Actually, this is a doc bug fix that should go in a separate PR and be merged and published right away.

Comment (Isammoc) [reply self-hosted-legacy/install/server/manual/index.md:534]: It's not a bug: it was valid.
I changed it only for coherence purpose.

Comment (slandelle) [reply self-hosted-legacy/install/server/manual/index.md:534]: OK, fine then.

**Review by slandelle — APPROVED** [review]

**Review by Obero — APPROVED** [review]

File: self-hosted-legacy/install/server/configuration/includes/frontline.conf [hunk 3]
```diff
@@ -105,5 +105,7 @@ frontline {
         # lastname: "family_name"
         # email: "email"
       }
+      # default role when user connect for the first time: none, viewer, tester, testAdmin and systemAdmin
```
Comment (stb13579) [inline self-hosted-legacy/install/server/configuration/includes/frontline.conf:108]:
```suggestion
      # default role when user connects for the first time: none, viewer, tester, testAdmin and systemAdmin
```

File: self-hosted-legacy/install/server/manual/index.md [hunk 4]
```diff
@@ -447,6 +447,8 @@ oidc {
     # lastname: "family_name" <9>
     # email: "email" <10>
   }
+  # default role when user connect for the first time: none, viewer, tester, testAdmin and systemAdmin
```
Comment (stb13579) [inline self-hosted-legacy/install/server/manual/index.md:450]:
```suggestion
  # default role when user connects for the first time: none, viewer, tester, testAdmin and systemAdmin
```

**Review by stb13579 — APPROVED** [review]
LGTM, just a small typo to fix.

**Review by tpetillot — APPROVED** [review]
